test/user.test.js

```diff
@@ -90,14 +92,13 @@ describe('User', function() {
     app.use(loopback.token({model: AccessToken}));
     app.use(loopback.rest());
 
-    User.create(validCredentials, function(err, user) {
-      if (err) return done(err);
-      validCredentialsUser = user;
-      User.create(validCredentialsEmailVerified, function(err, user) {
-        if (err) return done(err);
-        validCredentialsEmailVerifiedUser = user;
-        done();
-      });
+    // create 2 users: with and without verified email
+    return Promise.map(
+      [validCredentials, validCredentialsEmailVerified],
+      credentials => User.create(credentials)
+    ).then(users => {
+      validCredentialsUser = user = users[0];
+      validCredentialsEmailVerifiedUser = users[1];
```

Comment:
refactoring cb -> promises when creating users
assigning a shortcut `user` to use in tests rather than the heavy `validCredentialsUser`